ui/src/app/pages/admin/admin-institution.tsx

```diff
@@ -189,6 +193,13 @@ export const AdminInstitution = fp.flow(withNavigation)(
                 bodyStyle={styles.text}
                 headerStyle={{ ...styles.header, width: '5rem' }}
               />
+              <Column
+                field='requestAccessLink'
+                header='Custom Request Access URL'
+                body={this.renderRequestAccessLink}
```

Comment:
i wonder, (could be a question for UX team), but should this be clickable if it's not blank?